types/proper-lockfile/index.d.ts

```diff
@@ -1,13 +1,16 @@
-// Type definitions for proper-lockfile 3.0
+// Type definitions for proper-lockfile 4.1
 // Project: https://github.com/moxystudio/node-proper-lockfile
 // Definitions by: Nikita Volodin <https://github.com/qlonik>
 //                 Linus Unnebäck <https://github.com/LinusU>
```

Comment:
You could add your name to the list, if you wish

Reply:
Done.